frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -188,7 +188,7 @@ function CollectionSectionHeading({
         </SidebarLink>
         {currentUser.is_superuser && (
           <SidebarLink
-            icon={getCollectionIcon(PERSONAL_COLLECTIONS)}
+            icon={getCollectionIcon(PERSONAL_COLLECTIONS as Collection)}
```

Comment:
The `Collection` type does not allow `id=personal` but the `PERSONAL_COLLECTIONS` has such id. Wdyt about converting `frontend/src/metabase/entities/collections/constants.js` to TS?

Reply:
Good call. I've tried that and ran into exactly this issue. I'll see if I can put together another PR for this. We'd most likely have to change how we work with these "hardcoded" collections